### PR TITLE
Update the UI image and sync the contents of the readme and index

### PR DESCRIPTION
**Pull request by wpinrui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,25 @@
...
+
+To learn more about our team, head over to the [**About Us**](AboutUs.html) page.
+
+## User Guide
+
+If you are interested in using TutorAid, refer to the [**User Guide**](UserGuide.html).
+
+## Developer Guide
+
+If you are interested about developing TutorAid, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (JunWei3112) [inline README.md:21]: Need to update these links as well, since these do not work on the README

**Review by JunWei3112 — CHANGES_REQUESTED** [review]
Raised an issue regarding the links on the README. Once fixed, LGTM

File: README.md (hunk 1 above)
Comment (Preshita01) [inline README.md:21]: Minor nit-pick but can I check if it should be `interested in` or `interested about`? 😅

**Review by Preshita01 — APPROVED** [review]
LGTM just a minor comment

**Review by JunWei3112 — APPROVED** [review]
LGTM

**Pull request merged**